src/Cart.java

```diff
@@ -0,0 +1,6 @@
+import java.util.HashMap;
+
+public class Cart {
+  private HashMap<Long,String> purchases;
+  private long totalPrice;
```

Comment:
its better to store Product as a value in purchases
private long totalPrice; - subtotal